contrib/buildsystems/CMakeLists.txt

```diff
@@ -22,7 +27,7 @@ Instructions to run CMake manually:
 
 This will build the git binaries in contrib/buildsystems/out
 directory (our top-level .gitignore file knows to ignore contents of
-this directory).
+this directory). The project .sln and .vcproj files are also generated.
```

Comment:
You mean `.vcxproj`, with an `x`.

Reply:
BTW this is not the only place, there are also mentions of `.vcproj` in the commit message(s).

Reply:
Changed all the mistake `vcproj` to `vcxproj`.